scripts/collections-manager

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/env python
+import glob
+import os
+import subprocess
+import sys
+import tempfile
+import time
+from contextlib import contextmanager
+from importlib import import_module
+from pathlib import Path
+from typing import Generator, List
+
+import typer
+import yaml
+
+DEFAULT_ROOT = Path("~/src/github.com/PrefectHQ").expanduser()
+
+app = typer.Typer(help="Utilities for managing Prefect collections en masse")
+
+
+def prefect_collections() -> List[str]:
+    entries = []
+
+    for file in glob.glob("docs/integrations/catalog/*.yaml"):
+        if file.lower().endswith("/template.yaml"):
+            continue
+
+        with open(file) as f:
+            entries.append(yaml.safe_load(f))
+
+    entries = [e for e in entries if e["author"] == "Prefect"]
+    return [e["collectionName"] for e in entries]
+
+
+def ensure_repo(root: Path, collection: str) -> Path:
+    repo = root / collection
+    if not (repo / ".git").exists():
+        subprocess.check_call(
+            [
+                "git",
+                "clone",
+                f"git@github.com:PrefectHQ/{collection}.git",
+                str(repo),
+            ]
+        )
+
+    return repo
+
+
+def repositories(root: Path) -> Generator[Path, None, None]:
+    if not root.exists():
+        raise typer.BadParameter(f"Root directory {root} does not exist")
+
+    for collection in prefect_collections():
+        yield ensure_repo(DEFAULT_ROOT, collection)
```

Comment:
did you want `DEFAULT_ROOT` or `root` here?
```suggestion
        yield ensure_repo(root, collection)
```

Reply:
Woops great catch!